### PR TITLE
cleanup: Another small amount of import friction.

### PR DESCRIPTION
**Pull request by coryan**
I missed these problems in the last PR to reduce friction.

---
This change is [Reviewable](https://reviewable.io/reviews/googleapis/google-cloud-cpp/2800)

### CODE REVIEW

File: google/cloud/bigtable/tests/filters_integration_test.cc [hunk 1]
```diff
@@ -306,11 +306,13 @@ TEST_F(FilterIntegrationTest, CellsRowLimit) {
     auto inserted = actual.emplace(cell.row_key(), 0);
     inserted.first->second++;
   }
-  std::map<bigtable::RowKeyType, int> expected{{prefix + "/one-cell", 1},
-                                               {prefix + "/two-cells", 2},
-                                               {prefix + "/many", 3},
-                                               {prefix + "/many-columns", 3},
-                                               {prefix + "/complex", 3}};
+  using bigtable::RowKeyType;
```
Comment (devbww) [inline google/cloud/bigtable/tests/filters_integration_test.cc]: If we're going to use this to shorten the initializers (which I'm ambivalent on), should we also use it for the other instances of RowKeyType in this function?

File: google/cloud/bigtable/tests/filters_integration_test.cc [hunk 2]
```diff
@@ -329,9 +331,11 @@ TEST_F(FilterIntegrationTest, CellsRowOffset) {
     auto inserted = actual.emplace(cell.row_key(), 0);
     inserted.first->second++;
   }
-  std::map<bigtable::RowKeyType, int> expected{{prefix + "/many", 2},
-                                               {prefix + "/many-columns", 2},
-                                               {prefix + "/complex", 78}};
+  using bigtable::RowKeyType;
```
Comment (devbww) [inline google/cloud/bigtable/tests/filters_integration_test.cc]: Ditto.

File: google/cloud/bigtable/tests/filters_integration_test.cc (hunk 1 above)
Comment (coryan) [reply google/cloud/bigtable/tests/filters_integration_test.cc]: On second thought (thanks for asking) the longer name is fine. Fixed.

File: google/cloud/bigtable/tests/filters_integration_test.cc (hunk 2 above)
Comment (coryan) [reply google/cloud/bigtable/tests/filters_integration_test.cc]: Done.

**Pull request merged**